docs/WindowsCrossCompile.md

```diff
@@ -0,0 +1,44 @@
+# Cross-compiling Swift for Windows with `clang`.
+
+This document describes how to cross compile Swift on Windows on a non-Windows
+host. For more context on the status of Swift on Windows in general, see
+[the Windows doc](./Windows.md)
+
+## 1. Setup Visual Studio Environment Variables
+Building for Windows requires that the Visual Studio environment variables are
+setup similar to the values on Windows. Currently, the runtime has been tested
+to build against the Windows 10 SDK at revision 10.10.586.
+
+```bash
+# Visual Studio 2015 does not have VCToolsInstallDir, use VCINSTALLDIR's value
+export UCRTVersion=10.0.10586.0
+export UniversalCRTSdkDir=".../Windows Kits/10"
+export VCToolsInstallDir=".../Microsoft Visual Studio/2017/Community"
+```
+
+## 2. Setup `visualc` and `ucrt` modules
+The `visualc.modulemap` located at
+`swift/stdlib/public/Platform/visualc.modulemap` needs to be copied into
+`${VCToolsInstallDir}/include`. The `ucrt.modulemap` located at
+`swift/stdlib/public/Platform/ucrt.modulemap` needs to be copied into
+`${UniversalCRTSdkDir}/Include/${UCRTVersion}/ucrt`.
+
+## 3. Configure the runtime to be built with the just built clang
+Ensure that we use the tools from the just built LLVM and clang tools to build
+the Windows SDK. You will need to pass a few extra options to cmake via the
```

Comment:
Consistency: "clang" or "Clang"? "cmake" or "CMake"?

Reply:
I think the standard we should go for is `clang` and CMake